internal/server/proto/server.proto

```diff
@@ -493,6 +501,16 @@ message Project {
     // than the minimum will be ignored.
     string interval = 2;
   }
+
+  message AppPoll {
```

Comment:
Still a bit confused why some things are status polling and some are app polling and what other things might be polled in an app.

Reply:
Renamed everything from `App*` to `AppStatus*` to be extra clear what the settings are for